addons/GearSwap/refresh.lua

```diff
@@ -79,7 +79,7 @@ function load_user_files(job_id,user_file)
     end
     
     current_file = nil
-    gearswap_disabled = true
+    -- gearswap_disabled = true
```

Comment:
Remove this (and the others). No use for commented out code, that's what git is for :)

Reply:
Done, tho for some reason there's a bunch of other commented out lines already.